db/data/regions/_default/en.yml

```diff
@@ -146,75 +200,97 @@ en:
+            contains_emojis: City can't contain emojis
+            contains_mathematical_symbols: City can't contain mathematical symbols
             contains_restricted_characters: City can only contain letters, numbers,
               local characters, and special characters
-            contains_too_many_words: City cannot have more than %{word_count} words
-            contains_html_tags: City cannot contain HTML tags.
-            contains_url: City cannot contain a URL
-            unknown_for_zip: Enter a valid city for %{zip}
-            unknown_for_province: Enter a valid city for %{province}
-            unknown_for_zip_and_province: Enter a valid city for %{zip}, %{province}
-            unknown_for_address: City may be incorrect.
+            contains_too_many_words: City can't have more than %{word_count} words
+            contains_html_tags: City can't contain HTML
+            contains_url: City can't contain a URL
+            unknown_for_zip_instructional: Select a valid city for %{zip}
+            unknown_for_zip_informative: City doesn't match %{zip}
+            unknown_for_province_instructional: Enter a valid city for %{province}
+            unknown_for_province_informative: Province match %{province}
+            unknown_for_zip_and_province_instructional: Enter a valid city for %{zip}, %{province}
+            unknown_for_zip_and_province_informative: City may be incorrect
+            unknown_for_address: City may be incorrect
         province:
           label:
             default: Region
             optional: Region (optional)
           errors:
-            blank: Select a region
-            invalid: Enter a valid region
-            invalid_for_country: Enter a valid region for %{country}
-            unknown_for_city_and_zip: Enter a valid region for %{city}, %{zip}
-            unknown_for_city: Select a valid region for %{city}
-            unknown_for_zip: Select a valid region for %{zip}
-            unknown_for_address: Region may be incorrect.
+            blank_instructional: Select a region
+            blank_informative: Region is missing
+            invalid_instructional: Select a region
+            invalid_informative: Region is missing
+            invalid_for_country_instructional: Enter a valid region for %{country}
+            invalid_for_country_informative: Region doesn't match %{country}
+            unknown_for_city_and_zip_instructional: Enter a valid region for %{city}, %{zip}
+            unknown_for_city_and_zip_informative: Region doesn't match %{city}, %{zip}
+            unknown_for_city_informative: Select a valid region for %{city}
+            unknown_for_city_instructional: Region doesn't match %{city}
+            unknown_for_zip_instructional: Select a valid region for %{zip}
+            unknown_for_zip_informative: Region doesn't match %{zip}
+            unknown_for_address: Region may be incorrect
```

Comment:
Note to self: look into whether we should be using `select` or `enter` here, seems kind of random